src/debugger/Debugger.h

```diff
@@ -25,6 +25,8 @@
 #ifdef ESCARGOT_DEBUGGER
 namespace Escargot {
 
+#define ESCARGOT_DEBUGGER_MAX_STACK_TRACE_LENGTH 8
```

Comment:
Why did you limit `ESCARGOT_DEBUGGER_MAX_STACK_TRACE_LENGTH` value to 8?

Reply:
No special reason, it was something to start with. There are options here:
a) Unlimited trace (may cost a lot of memory)
b) Compile time limited (current approach)
c) Dynamic limit configuration (from API or ES code)
Which one do you prefer? I think c) can be done later in a separate patch.

Reply:
I see. current method (b) looks good to me :)